A temporary workaround for the too slow start up in Wayland mode (bsc#1247542)
---
 xfce4-session/xfsm-manager.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/xfce4-session/xfsm-manager.h
+++ b/xfce4-session/xfsm-manager.h
@@ -40,7 +40,8 @@ G_BEGIN_DECLS
 #define DIE_TIMEOUT (7 * 1000)
 #define SAVE_TIMEOUT (60 * 1000)
 #define STARTUP_TIMEOUT (8 * 1000)
-#define STARTUP_TIMEOUT_WAYLAND (8 * 1000)
+// FIXME: reduced to 500ms for Wayland, otherwise it takes ages at startup
+#define STARTUP_TIMEOUT_WAYLAND 500
 #define RESTART_RESET_TIMEOUT (5 * 60 * 1000)
 #define XFSM_CHOOSE_LOGOUT 0
 #define XFSM_CHOOSE_LOAD 1
